comp/core/status/statusimpl/status_api_endpoints.go

```diff
@@ -48,13 +44,38 @@ func (s *statusImplementation) getStatus(w http.ResponseWriter, r *http.Request,
 		buff, err = s.GetStatus(format, verbose)
 	}
 
+	if len(buff) != 0 {
+		// scrub status output
+		s := scrubber.DefaultScrubber
+		if format == "json" {
+			buff, err = s.ScrubJSON(buff)
+		} else {
+			buff, err = s.ScrubBytes(buff)
+		}
+	}
+
 	if err != nil {
+		var errorMsg string
+		var scrubbedMsg []byte
+		var scrubOperationErr error
+
 		if format == "text" {
-			http.Error(w, s.log.Errorf("Error getting status. Error: %v.", err).Error(), http.StatusInternalServerError)
```

Comment:
Do we really want to keep the `.` at the end of the string ?

Reply:
You could even replace it with just `status: %v` 😄